drizzlepac/hlautils/product.py

```diff
@@ -154,9 +169,17 @@ def generate_footprint_mask(self):
         """
         footprint = cell_utils.SkyFootprint(self.meta_wcs)
         exposure_names = [element.full_filename for element in self.edp_list]
-        footprint.build(exposure_names)
+        footprint.build(exposure_names, scale=True)
         self.mask = footprint.total_mask
 
+        # Compute footprint-based SVM-specific keywords for product image header
+        good_pixels = self.mask > 0
+        self.mask_kws['NPIXFRAC'][0] = good_pixels.sum()
```

Comment:
This looks to be the sum of pixels with good data versus a fraction (good_pixels/all_pixels).

Reply:
Also fixed in 871bdef.